src/components/Footer/index.tsx

```diff
@@ -10,6 +10,8 @@ interface Props {
   hideNewsletter?: boolean;
 }
 
+const CURRENT_YEAR = process.env.CURRENT_YEAR || new Date().getFullYear();
```

Comment:
I added this env variable in case the user has a custom/wrong year on this pc. If that's the case we could use the time from our server